server/src/services/asset.service.ts

```diff
@@ -224,6 +228,13 @@ export class AssetService {
     return this.assetRepository.getAllByDeviceId(auth.user.id, deviceId);
   }
 
+  public async getAllAssets(auth: AuthDto, dto: AssetSearchDto): Promise<AssetResponseDto[]> {
+    const userId = dto.userId || auth.user.id;
+    await this.access.requirePermission(auth, Permission.TIMELINE_READ, userId);
+    const assets = await this.assetRepositoryV1.getAllByUserId(userId, dto);
```

Comment:
I am not a big fan of having asset repo v1 stuff in here tbh.

Reply:
The mobile app just stopped using it, yeah. I think it is still being used in production today though

Reply:
Then it is fine removing it with this release ;)